cluster/manifests/kube-downscaler/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+{{ if neq .Environment "production" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kube-downscaler
+  namespace: kube-system
+  labels:
+    application: kube-downscaler
+    version: v0.3
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      application: kube-downscaler
+  template:
+    metadata:
+      labels:
+        application: kube-downscaler
+        version: v0.3
+    spec:
+      serviceAccountName: system
+      containers:
+      - name: downscaler
+        # see https://github.com/hjacobs/kube-downscaler/releases
+        image: hjacobs/kube-downscaler:0.3
+        args:
+          # run every 3 minutes
+          - --interval=180
+          - --exclude-namespaces=kube-system,visibility
+          - "--default-uptime={{ .ConfigItems.downscaler_default_uptime }}"
+        resources:
+          limits:
+            memory: 100Mi
```

Comment:
Can we please avoid over-committing :)

Reply:
@mikkeloscar done, fixed --- this was not intended and I copied the manifest from https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/external-dns/deployment.yaml . We should probably once clean up all manifests if we want to avoid overcommit.